table/block_based_table_reader.cc

```diff
@@ -255,7 +277,7 @@ class PartitionIndexReader : public IndexReader, public Cleanable {
       const bool is_index = true;
       s = table_->MaybeLoadDataBlockToCache(prefetch_buffer.get(), rep, ro,
                                             handle, compression_dict, &block,
-                                            is_index);
+                                            is_index, nullptr);
```

Comment:
maybe let's put TODOs to support this for partitioned index and filter blocks -- I imagine we are bumping a lot of counters for those when enabled